### DISCUSSION TITLE
Changing specific blocks to be other blocks

### DISCUSSION THREAD

**Discussion by vytskalt** [rutgerkok/WorldGeneratorApi#41] [Q&A] [answered]
Hello everyone,

I need to change all lava in the nether to another block. Is this possible?

**Comment by tastybento** [comment, marked as answer]
Yes. You have to override the getTerrianSettings method and define what the water block is going to be. See my code https://github.com/BentoBoxWorld/Boxed/blob/develop/src/main/java/world/bentobox/boxed/generators/NetherGenerator.java

Around Line 81

**Reply by vytskalt** [reply]
Thank you so much!
